README.rst

```diff
@@ -252,14 +308,16 @@ Use python-lambda-local
 
     python-lambda-local tests/test.py tests/event.json -a 'arn:aws:lambda:us-east-1:accountId:function:functionNamePython:$LATEST'
 
-Packaging
-~~~~~~~~~
+
+Publish a new version
+^^^^^^^^^^^^^^^^^^^^^^^
+If you would like to publish a new version, run the following command in your command line to install a new Python package (build a wheel): 
 
 .. code::
 
     python setup.py bdist_wheel --universal
-
+    
 License
-~~~~~~~
+^^^^^^^^
+Apache Software License v2. Copyright © 2014-2019 SignalFx    
```

Comment:
We made it to 2020 ;)

Reply:
This has been updated.

Reply:
We (Joe and I) had to update our copyright statements to say Splunk. To be exact:

(c) Copyright 2020 Splunk, Inc. 

So if this type of copyright statement is in the readme files, they all need to be updated. (Fine with me if it says 2014-2020.) Need a jira ticket and should do it ASAP, obviously.

Reply:
Thanks for clarification. I updated the copyrights to say Splunk.